stories/setup/webpack.config.babel.js

```diff
@@ -53,6 +53,6 @@ module.exports = {
     alias: {
       normalize: 'normalize.css/normalize.css',
     },
-    modules: [path.resolve('./src'), 'node_modules'],
+    modules: [path.resolve('./src'), path.resolve('./stories'), 'node_modules'],
```

Comment:
It's not a good idea to resolve `./stories` like this because [`src/ui`](https://github.com/mozilla/addons-frontend/tree/master/src/ui) will overshadow `stories/ui`.

Maybe we could solve it by adjusting the directory layout?

```
stories/setup/
stories/index.js
stories/stories/ <-- everything would now go here.
```

However, I don't see why you need to resolve the path to `./stories` in the first place. Is that just something you want to use in the future?

If you can, maybe try getting things working with relative imports first so we can see how annoying it is.

Reply:
actually it was working as expected but linting and flow were complaining about this when I ran those checks : / 

I'll remove what I've added above as you suggest tho w/ relative paths for now